### ISSUE TITLE
Even when I do -y it still eventually requires user input on longer requests

### ISSUE THREAD

**Issue by wakywayne** [KillianLucas/open-interpreter#981]
### Describe the bug

Lets say I have it make a slightly larger then simple program / file. It will start by not need my input then after a few minutes will eventually not continue without my input. It seems it is usually right before starting to build the program.

### Reproduce

Ask it to build a simple on file application that uses some oop classes like a tic tac toe game or something and use the interpreter -y command to run open interpreter.

### Expected behavior

I would expect to have the interpreter not need me to ever put in input until it is done with the project.

### Open Interpreter version

current as of 1/30/24

### Python version

3.1

### Operating System name and version

Unbuntu 22

**Comment by MikeBirdTech** [comment]
Hi @wakywayne 
When it stops, does it ask you to provide a `y` to continue or does it ask other questions? Have you tried with the latest release?
Thanks!

**Comment by interpreterwork** [comment]
Thank you for reporting this and for helping build the classic Python version of Open Interpreter. This issue concerns the earlier Python/LiteLLM/Jupyter product; this repository now tracks the Rust Open Interpreter built on Codex. We are closing the legacy report here so current product tracking starts clean. If you still want to pursue it, the community-maintained classic fork is https://github.com/endolith/open-interpreter. If the same problem affects the current Interpreter release, please open a new issue with the current template and include the version, provider, model, wire API, harness, platform, and reproduction.
